packages/script/src/script.ts

```diff
@@ -120,4 +120,23 @@ export class Script<TInput extends Array<any>, TOutput> extends AbstractScript {
 
     return this;
   }
+
+  /**
+   *
+   * @param deployer - The account used to pay the deployment costs.
+   * @returns The _blobId_ and a _waitForResult_ callback that returns the deployed predicate
+   * once the blob deployment transaction finishes.
+   *
+   * The returned loader script will have the same configurable constants
+   * as the original script which was used to generate the loader script.
+   */
+  deploy<T = this>(deployer: Account) {
+    return deployScriptOrPredicate<T>({
+      deployer,
+      abi: this.interface.jsonAbi,
+      bytecode: this.bytes,
+      loaderInstanceCallback: (loaderBytecode, newAbi) =>
+        new Script(loaderBytecode, newAbi, this.account) as T,
+    });
```

Comment:
Even though `deployer` works as a parameter name here, I'd rather use `account` since it's clearer and more in line with the usual terms in the TS SDK.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3389/commits/843567397d6275918c516eb4b554a9eeb56923be